hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineManager.java

```diff
@@ -61,8 +61,8 @@ Pipeline createPipeline(ReplicationType type, ReplicationFactor factor,
       ReplicationFactor factor, Pipeline.PipelineState state);
 
   List<Pipeline> getPipelines(ReplicationType type, ReplicationFactor factor,
-      Pipeline.PipelineState state, Collection<DatanodeDetails> excludeDns,
-      Collection<PipelineID> excludePipelines);
```

Comment:
Can we keep the existing one parameter as Collection which allows passing Set here?